docs/resources/identity_agency.md

```diff
@@ -27,6 +28,25 @@ resource "huaweicloud_identity_agency" "agency" {
 }
 ```
 
+### Delegate the operation authority of the resources in the account to Huaweicloud service
```

Comment:
Delegate a cloud service to access your resources in other cloud services.

Reply:
thanks for suggestion, it's updated at document.